packages/cli/src/platform/accelerate/enable.ts

```diff
@@ -17,67 +12,112 @@ export class Enable implements Command {
+            databaseLinkCreate(input: $input) {
+              __typename
+              ... on Error {
+                message
+              }
+              ... on DatabaseLink {
+                id
+              }
+            }
+          }
+        `,
+        variables: {
+          input: {
+            environmentId,
+            connectionString,
+          },
+        },
       },
     })
-    if (accelerateSetupPayload.error) {
-      throw new Error(accelerateSetupPayload.error.message)
-    }
-    if (apikey) {
-      const payload = await platformRequestOrThrow<{
-        data: {
-          tenantAPIKey: string
+    const { serviceTokenCreate } = await requestOrThrow<
+      {
+        accelerateEnable: {}
+        serviceTokenCreate?: {
+          value: string
         }
-        error: null | { message: string }
-      }>({
-        token,
-        path: `/${workspace}/${project}/settings/api-keys/create`,
-        route: '_app.$organizationId_.$projectId.settings.api-keys.create',
-        payload: {},
-      })
-      if (payload.error?.message) {
-        throw new Error(payload.error.message)
+      },
+      null,
+      {
+        accelerateEnableInput: { databaseLinkId: string }
+        serviceTokenCreateInput: { environmentId: string }
+        withServiceToken: boolean
       }
-      return successMessage(
-        `Accelerate enabled. Use this generated API key in your Accelerate connection string to authenticate requests:\n\n${generateConnectionString(
-          payload.data.tenantAPIKey,
-        )}\n\nFor more information, check out the Getting started guide here: ${link(
-          'https://pris.ly/d/accelerate-getting-started',
-        )}`,
+    >({
+      token,
+      body: {
+        query: /* GraphQL */ `
+          mutation (
+            $accelerateEnableInput: MutationAccelerateEnableInput!
+            $serviceTokenCreateInput: MutationServiceTokenCreateInput!
+            $withServiceToken: Boolean!
+          ) {
+            accelerateEnable(input: $accelerateEnableInput) {
+              __typename
+              ... on Error {
+                message
+              }
+            }
+            serviceTokenCreate(input: $serviceTokenCreateInput) @include(if: $withServiceToken) {
+              __typename
+              ... on Error {
+                message
+              }
+              ... on ServiceTokenWithValue {
+                value
+              }
+            }
+          }
+        `,
+        variables: {
+          withServiceToken,
+          accelerateEnableInput: { databaseLinkId: databaseLinkCreate.id },
+          serviceTokenCreateInput: { environmentId },
+        },
+      },
+    })
+
+    const gettingStartedUrl = link('https://pris.ly/d/accelerate-getting-started')
+
+    if (serviceTokenCreate) {
+      return messages.success(
+        `Accelerate enabled. Use this generated API key in your Accelerate connection string to authenticate requests:\n` +
```

Comment:
```suggestion
        `Accelerate enabled. Use this Accelerate connection string to authenticate requests:\n` +
```

Reply:
From what I see `generateConnectionString` actually returns the connection string and not just the API key, so I think it makes sense to update the message